lib/pip.js

```diff
@@ -609,6 +598,8 @@ async function installRequirementsIfNeeded(
   // Our source requirements, under our service path, and our module path (if specified)
   const fileName = path.join(servicePath, modulePath, options.fileName);
 
+  await pyprojectTomlToRequirements(modulePath, pluginInstance);
```

Comment:
Is there a reason why it's being called as a part of `installRequirementsIfNeeded`? I think it's not the best place to call this logic as if I'm not mistaken, right now, for case where the individual packaging is not used, it will be called twice (once as a part of logic in `index.js` and second time as a part of logic here which seems unnecessary) - could you confirm that's the case?

Reply:
Maybe it would be better to first iterate over the modules and generate the `requirement.txt` files for each of them and then proceed to installation?

Reply:
I will look into it

Reply:
It turns out that the call in the top-level sequence was no longer necessary.  I removed it and confirmed that all tests pass.